### PR TITLE
Release 2017 39 no RN changes

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Weekly changes are all doc-only.

### Partially Resolves:
 [OKTA-142434](https://oktainc.atlassian.net/browse/OKTA-142434)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
This is my first push from weekly, so @briandemers-okta can you verify this looks right to you?
Instructions say to base this PR on origin/weekly, that's why the source PRs are in this PR I believe.

File: Gemfile.lock [hunk 1]
```diff
@@ -189,7 +189,6 @@ GEM
       gemoji (~> 3.0)
       html-pipeline (~> 2.2)
       jekyll (>= 3.0)
-    json (2.1.0)
```
Comment (bdemers) [inline Gemfile.lock]: What OS are you on? This file keep changing. We can deal with it outside of this issue, but I wanted to start collecting data.

File: _source/_code/php/quickstart-signin-widget.md [hunk 2]
```diff
@@ -225,7 +226,7 @@ $jws->verify($jwk, 'RS256');
 Each public key is identified by a `kid` attribute, which corresponds with the `kid` claim in the [Access Token header](/docs/api/resources/oauth2.html#token-authentication-methods).
 
 The access token is signed by an RSA private key, and we publish the future signing key well in advance.
-However, in an emergency situation you can still stay in sync with Okta&#8217;s key rotation. Have your application check the `kid`, and if it has changed and the key is missing from the local cache, check the `jwks_uri` value in the [authorization server metadata](/docs/api/resources/oauth2.html#retrieve-authorization-server-metadata) and you can go back to the [jwks uri](/docs/api/resources/oauth2.html#get-keys) to get keys again from Okta
```
Comment (bdemers) [inline _source/_code/php/quickstart-signin-widget.md]: Are these PHP changes expected to have been updated i `weekly` and not a directly against `source`
cc: @bretterer

**Comment by mystiberry-okta** [comment]
@jmelberg-okta or @jakubvul can also review.

File: Gemfile.lock (hunk 1 above)
Comment (mystiberry-okta) [reply Gemfile.lock]: This is from a file someone else checked in (I think, I remember looking at it before). I have had so much trouble that I don't check in Gemfile.lock anymore (except by mistake). I'm in VM linux (albatross), goal is to get out next month.

Comment (bdemers) [reply Gemfile.lock]: no worries, I end up with the same 'Gemfile.lock' as this diff.

File: _source/_code/php/quickstart-signin-widget.md (hunk 2 above)
Comment (mystiberry-okta) [reply _source/_code/php/quickstart-signin-widget.md]: Oh, yes, I had just randomly seen the missing period and added it. I try not to mix things up like that.

File: Gemfile.lock (hunk 1 above)
Comment (mystiberry-okta) [reply Gemfile.lock]: @bdemers , Can you click Approve in the Review changes dialog? on source I need an official review to check in.

File: _source/_code/php/quickstart-signin-widget.md (hunk 2 above)
Comment (bdemers) [reply _source/_code/php/quickstart-signin-widget.md]: If it is expected, I'm +1

**Pull request merged**